### PR TITLE
Fix/graphql inconsistencies

### LINKED ISSUE #46: tickets_inc_discount_price
Alex says we should add `tickets_discounted_price` to the response and then `sub_total` should be reserved for the future when we have extra stuff (like programs and :popcorn: ).

DoD: 
Add extra field to ticket price break down serializer which also gives the subtotal called `tickets_discounted_price`

### LINKED ISSUE #63: Add custom image model
I think this is a good idea but not 100% yet.
Somehow extend existing image model. Then just create a schema for that new model and remove all the nasty mixin stuff.
Then we can add extra stuff onto the image model as required.

### LINKED ISSUE #98: image full URL

### PR DESCRIPTION
**Pull request by streth11**
fixes many of the inconsistencies with the front-end

closes #98 
closes #63 
closes #46 

TODO:

- [x] Test bookings orderBy filter
- [x] Test IdInputField with parameters
- [x] Image Model

### CODE REVIEW

**Comment by streth11** [comment]
wait for kit's stuff before merge

**Comment by walkerkit** [comment]
Are we still brining this in before the refactor?

File: uobtheatre/bookings/schema.py
Comment (JElgar) [inline uobtheatre/bookings/schema.py:70]: There are a few options for this which I think might be a bit betterer. One option is to set the discount node queryset but I think the ideal is to somehow set the "default" filtering.

**Review by JElgar — CHANGES_REQUESTED** [review]

**Comment by JElgar** [comment]
@walkerkit lets branch off this and then start the refactor since there will be very few changes for this branch before its merged.

File: uobtheatre/bookings/schema.py
Comment (streth11) [reply uobtheatre/bookings/schema.py:70]: james this fix is beyone my level, sooo its all yours bby

**Review by JElgar — APPROVED** [review]
LGTM

**Pull request merged**
